Automattic/wp-calypso
# Desktop App: Update app icons

Fixes #92825

## Proposed Changes

* Update the Windows, Linux, and MacOS app icons


### MacOS
| Before | After |
|--------|--------|
| ![image](https://github.com/user-attachments/assets/4a83b109-a5eb-40b3-a8cd-37ccc305ffef) | ![image](https://github.com/user-attachments/assets/61b7c1f1-d35c-4c55-8970-160053634f50) |

### Windows
| Before | After |
|--------|--------|
| ![image](https://github.com/user-attachments/assets/ff45f592-b52f-4fb6-9057-f8de77c3c469) | ![image](https://github.com/user-attachments/assets/f7450ec9-989d-4c37-8b72-42a12f799298) |

### Linux
| Before | After |
|--------|--------|
| ![image](https://github.com/user-attachments/assets/4403d03c-a9bd-4c90-bab3-11e6b7dd744c) | ![image](https://github.com/user-attachments/assets/f7450ec9-989d-4c37-8b72-42a12f799298) |

## Why are these changes being made?


* To bring the icons inline with the latest WordPress.com branding.

## Testing Instructions



* From the `desktop` directory, run `yarn run build-desktop`
* Go to the `desktop/release` folder and check that the built app files use the correct icons.

## Pre-merge Checklist
